validator/accounts/accounts_enable_disable.go

```diff
@@ -182,7 +182,7 @@ func EnableAccountsCli(cliCtx *cli.Context) error {
 		}); err != nil {
 			return err
 		}
-		log.WithField("publicKeys", allAccountStr).Info("Accounts enabled")
+		log.WithField("publicKeys", allAccountStr).Info("Accounts are enabled. PLEASE RESTART YOUR VALIDATOR NODE for those updates to take effect")
```

Comment:
With the current approach of detecting a change for keymanageropts.json, I am able to hot-reload after disabling an account but I am NOT able to hot-reload after re-enabling an account.
This can be explained by the fact that keymanageropts.json contains a list of disabled_public_keys, removing a key from that list can be picked up by the watcher, but we won't know which one has been removed.